mailpoet/assets/js/src/automation/integrations/mailpoet/components/form-token-field/index.tsx

```diff
@@ -0,0 +1,81 @@
+import { FormTokenField as WpFormTokenField } from '@wordpress/components';
+import { uniq } from 'lodash';
+
+export type FormTokenItem = {
+  id: number | string;
+  name: string;
+};
+
+export type FormTokenFieldProps = Omit<
+  WpFormTokenField.Props,
+  'value' | 'suggestions' | 'onChange'
```

Comment:
The `value` prop doesn't seem to by overwritten. Do we need to `omit` it?

Reply:
Issue solved via [db83b9b](https://github.com/mailpoet/mailpoet/pull/4371/commits/db83b9bbc1755f2c31919ec4036b3b37bc830439). In [db83b9b](https://github.com/mailpoet/mailpoet/pull/4371/commits/db83b9bbc1755f2c31919ec4036b3b37bc830439) I align our props with the original FormTokenField.Props. I used `selected` where FormTokenField uses `value`. Now, we use `value` as well.